link_linux.go

```diff
@@ -1150,8 +1150,8 @@ func (h *Handle) linkModify(link Link, flags int) error {
 		native.PutUint32(b, uint32(base.ParentIndex))
 		data := nl.NewRtAttr(unix.IFLA_LINK, b)
 		req.AddData(data)
-	} else if link.Type() == "ipvlan" {
-		return fmt.Errorf("Can't create ipvlan link without ParentIndex")
+	} else if (link.Type() == "ipvlan") || (link.Type() == "ipoib") {
```

Comment:
It's not C, these brackets are not needed ;)
Please replace that with:
```go
} else if link.Type == "ipvlan" || link.Type() == "ipoib" {
```

Reply:
Mainly added brackets as it was clearer for me to read , and gofmt didn't seem bothered by it :)
Anyway, will remove as i see other places in the project dont use em.